Jenkinsfile

```diff
@@ -311,7 +311,7 @@ def Tasks = [
   "tools": '''
     setJavaVersion $java
     npm install &&
-    sbt ++$scala ir$v/test logging$v/compile linkerInterface$v/compile \
+    sbt ++$scala ir$v/test logging$v/compile linkerInterface$v/test \
```

Comment:
The linkerInterface tests also need to run in JS. (look for `linkerInterfaceJS$v`)

Reply:
Fixed in b58a8673c6b5bd196a2e9af6d86353b59a91ec12